rootfs/standard/usr/share/mynode_apps/itchysats/scripts/install_itchysats.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+source /usr/share/mynode/mynode_device_info.sh
+source /usr/share/mynode/mynode_app_versions.sh
+
+set -x
+set -e
+
+echo "==================== INSTALLING APP ===================="
+
+# The current directory is the app install folder and the app tarball from GitHub
+# has already been downloaded and extracted. Any additional env variables specified
+# in the JSON file are also present.
+
+apt-get -y install cmake
+mynode-install-extra rust
```

Comment:
The apt-get install cmake may not work here. Try adding it to mynode_post_upgrade.sh in the section where all the other stuff is installed via apt-get.

Yes, that's the right way to do it, but I did just find a bug. I don't think that script currently works as the bitcoin user. I think I just worked out a fix.

Reply:
Since we specify `"linux_user": "root"` in `itchysats.json` it does work, but maybe that's misconfigured.

In any case, I think we need to be `root` for this to work anyway:

```
sudo install -g root -o root target/release/taker /usr/bin/itchysats
```

Do you think there is a better way to install the app?

Reply:
After install, does it really have to copy the binary to /usr/bin? Most other apps set the directory of their app folder and run as the linux_user that is specified in the JSON file.

Reply:
And they do that by setting things in their service file - set the WorkingDir, User, Group, etc... and then to run itchysats, you can execute the binary that is in the folder within the workingdir.